paddle/fluid/operators/setitem_op.h

```diff
@@ -0,0 +1,199 @@
+//   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <algorithm>
+#include <string>
+#include <vector>
+
+#include <utility>
+#include "paddle/fluid/framework/eigen.h"
+#include "paddle/fluid/framework/op_registry.h"
+#include "paddle/fluid/operators/assign_value_op.h"
+#include "paddle/fluid/operators/elementwise/elementwise_op_function.h"
+#include "paddle/fluid/platform/enforce.h"
+
+namespace paddle {
+namespace operators {
+
+using Tensor = framework::Tensor;
+
+inline const char* GetValueName(framework::proto::VarType::Type data_type,
```

Comment:
Not a good function style because the output `value_name` is at both the return and the parameter.

I searched the place where you were using this function, you could write:
```
inline std::string GetValueName(framework::proto::VarType::Type data_type)
```
Then use it like:
```
std::string value_name = GetValueName(dtype)
CopyVecotorToTensor<T>(value_name.c_str(), &value_t, ctx);
```

Reply:
Thanks, done